topi/tests/python/test_topi_conv2d_nchw.py

```diff
@@ -1,12 +1,14 @@
 """Example code to do convolution."""
```

Comment:
Does this only use workloads that exist in current popular networks e.g., (ResNet-50, VGG, Superresolution)? If so, we may just wan to add some weird shapes to test the fallback mechanism.

Reply:
I don't load pre-tuned parameters in this test. All these workloads will use fallback configs.